src/routes/api/discord/[userId].ts

```diff
@@ -0,0 +1,30 @@
+import { Routes } from 'discord-api-types/v10'
+import type { APIGuildMember } from 'discord-api-types/v10'
+import { api } from '$lib/discord'
+import type { RequestHandler } from '@sveltejs/kit'
```

Comment:
minor nit, can we re-order these?

```suggestion
import type { APIGuildMember } from 'discord-api-types/v10'
import { api } from '$lib/discord'
import type { RequestHandler } from '@sveltejs/kit'
```

And would you be open to adding this lint rule to ESLint? https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md

```
'import/order': [
      2,
      {
        groups: ['builtin', 'external', 'parent', 'sibling', 'index', 'type'],
        'newlines-between': 'never',
      },
    ],
```